src/Bicep.Core/TypeSystem/ErrorSymbolExtensions.cs

```diff
@@ -6,7 +6,7 @@ namespace Bicep.Core.TypeSystem
 {
     public static class ErrorSymbolExtensions
```

Comment:
Can you rename this class (plus any other references to `ErrorSymbol`) to be consistent?

Reply:
Done,